python/tvm/driver/tvmc/model.py

```diff
@@ -0,0 +1,357 @@
+            records can be stored.
+        """
+        return self._tmp_dir.relpath("tuning_records")
+
+    def export_package(
+        self,
+        executor_factory: GraphExecutorFactoryModule,
+        package_path: Optional[str] = None,
+        cross: Optional[Union[str, Callable]] = None,
+    ):
+        """Save this TVMCModel to file.
+
+        Parameters
+        ----------
+        executor_factory : GraphExecutorFactoryModule
+            The compiled graph factory that will be used to generate the output package.
+        package_path : str, None
+            Where the model should be saved. Note that it will be packaged as a .tar file.
+            If not provided, the package will be saved to a generically named file in tmp.
+        cross : str or callable object, optional
+            Function that performs the actual compilation.
+        """
+        if package_path is None:
+            package_path = self._tmp_dir.relpath("package.tar")
+        export_model_library_format(executor_factory, package_path, cross=cross)
+        self.package_path = package_path
+
+    def summary(self):
+        """ Print the IR corressponding to this model."""
+        print(self.mod)
+
+
+class TVMCPackage(object):
+    """Load a saved TVMCPackage from disk.
+
+    Parameters
+    ----------
+    package_path : str
+        The path to the saved TVMCPackage that will be loaded.
+    """
+
+    def __init__(self, package_path: str):
+        self._tmp_dir = utils.tempdir()
+        self.import_package(package_path)
+
+    def import_package(self, package_path: str):
+        """Load a TVMCPackage from a previously exported TVMCModel.
+
+        Parameters
+        ----------
+        package_path : str
+            The path to the saved TVMCPackage.
+        """
+        temp = self._tmp_dir
+        t = tarfile.open(package_path)
+        t.extractall(temp.relpath("."))
+
+        metadata_path = "metadata.json"
+        with open(temp.relpath(metadata_path), "r") as metadata_file:
+            metadata = json.load(metadata_file)
+        self.name = metadata["model_name"]
+        self.target = metadata["target"]
+        self.runtimes = metadata["runtimes"]
+
+        parameter_path = os.path.join("parameters", f"{self.name}.params")
+        with open(temp.relpath(parameter_path), "rb") as param_file:
+            self.params = bytearray(param_file.read())
+
+        graph_path = os.path.join("runtime-config", "graph", "graph.json")
+        with open(temp.relpath(graph_path), "r") as graph_file:
+            self.graph = graph_file.read()
+
+        ir_path = "relay.txt"
+        with open(temp.relpath(ir_path), "r") as relay_file:
+            self.mod = relay_file.read()
+
+        self.lib_path = temp.relpath(f"{self.name}.so")
+
+    def summary(self):
+        print(self.mod)
+
+
+class TVMCResult(object):
+    """Create a convenience wrapper around the output of tvmc.run
```

Comment:
I think this docstring is actually for `__init__`. https://numpydoc.readthedocs.io/en/latest/format.html#documenting-classes

or, change "Parameters" to "Attributes" here and adjust the one-liner to describe the class rather than the function.